cpp/pmft/PMFTXY2D.cc

```diff
@@ -32,11 +32,11 @@ PMFTXY2D::PMFTXY2D(float x_max, float y_max, unsigned int n_x, unsigned int n_y)
     m_pcf_array.prepare({n_x, n_y});
 
     // Construct the Histogram object that will be used to keep track of counts of bond distances found.
-    util::Histogram::Axes axes;
+    BHAxes axes;
```

Comment:
Does some of this belong in the parent PMFT class? I see it's repeated.

Reply:
I'm not sure exactly which lines you're referring to, but the construction of the histogram has to be specific to a given PMFT based on the provided coordinates.